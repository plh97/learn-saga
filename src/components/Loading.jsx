import React from 'react'
export default () => (
  <div className="loading">
    加载中...
  </div>
)