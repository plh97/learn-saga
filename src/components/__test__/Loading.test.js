import React from 'react'
import Loading from '../Loading'
import renderer from 'react-test-renderer'

// LabelItem 组件 未选中 状态
describe('Loading snapshot rending test', () => {
  it('renders correctly testing', () => {
    const component = renderer
      .create(<Loading />)
      .toJSON()
    expect(component).toMatchSnapshot()
  })
})
