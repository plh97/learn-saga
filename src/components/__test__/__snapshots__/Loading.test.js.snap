// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot rending test renders correctly testing 1`] = `
<div
  className="loading"
>
  加载中...
</div>
`;
